build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url 'https://jitpack.io' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
Would `7.2.0` also work? I am on latest version of IntelliJ and this 7.3.1 is not yet supported. @CAMOBAP

Reply:
I assume that it will work, let me update the PR

Reply:
I will try with 2022.3 RC. Based on the changelog this gradle 7.3.1 should be supported.

Reply:
Let's downgrade to 7.2.2 for now since IDE support for 7.3.1 is only in beta.